tensorflow_datasets/text/snli.py

```diff
@@ -0,0 +1,155 @@
+	Author = {Bowman, 
+              Samuel R. and Angeli, 
+              Gabor and Potts, Christopher, 
+              and Manning, Christopher D.
+            },
+	Booktitle = {Proceedings of the 2015 Conference on Empirical Methods in Natural Language Processing (EMNLP)},
+	Publisher = {Association for Computational Linguistics},
+	Title = {A large annotated corpus for learning natural language inference},
+	Year = {2015}
+}
+"""
+
+_DESCRIPTION = """\
+The SNLI corpus (version 1.0) is a collection of 570k human-written English sentence
+pairs manually labeled for balanced classification with the labels entailment, contradiction,
+and neutral, supporting the task of natural language inference (NLI), also known as recognizing
+textual entailment (RTE). We aim for it to serve both as a benchmark for evaluating representational
+systems for text, especially including those induced by representation learning methods,
+as well as a resource for developing NLP models of any kind.
+"""
+
+
+class SNLIConfig(tfds.core.BuilderConfig):
+  """BuilderConfig for SNLI."""
+
+  @api_utils.disallow_positional_args
+  def __init__(self, text_encoder_config=None, **kwargs):
+    """BuilderConfig for SNLI.
+
+    Args:
+      text_encoder_config: `tfds.features.text.TextEncoderConfig`, configuration
+        for the `tfds.features.text.TextEncoder` used for the features feature.
+      **kwargs: keyword arguments forwarded to super.
+    """
+    super(SNLIConfig, self).__init__(**kwargs)
+    self.text_encoder_config = (
+        text_encoder_config or tfds.features.text.TextEncoderConfig())
+
+
+class SNLI(tfds.core.GeneratorBasedBuilder):
+  """SNLI: The Stanford Natural Language Inference (SNLI) Corpus. Version 1.0"""
+
+  BUILDER_CONFIGS = [
+      SNLIConfig(
+          name="plain_text",
+          version="1.0.0",
+          description="Plain text",
+      ),
+  ]
+
+  def _info(self):
+    return tfds.core.DatasetInfo(
+        builder=self,
+        description=_DESCRIPTION,
+        features=tfds.features.FeaturesDict({
+            "text":
+                tfds.features.Text(
+                    encoder_config=self.builder_config.text_encoder_config),
+            "hypothesis":
+                tfds.features.Text(
+                    encoder_config=self.builder_config.text_encoder_config),
+            "judgements":
+                tfds.features.Text(
+                    encoder_config=self.builder_config.text_encoder_config),
+        }),
+        # No default supervised_keys (as we have to pass both text
+        # and hypothesis as input).
+        supervised_keys=None,
+        urls=["https://nlp.stanford.edu/projects/snli/"],
+        citation=_CITATION,
+    )
+
+  def _vocab_text_gen(self, filepath):
+    for ex in self._generate_examples(filepath):
+      yield " ".join([ex["text"], ex["hypothesis"], ex["judgements"]])
+
+  def _split_generators(self, dl_manager):
+
+    downloaded_dir = dl_manager.download_and_extract(
```

Comment:
Yo shouldn't be written URL directly here. Define URL above.

Reply:
Honestly, I don't think it matters much. But I agree having them as constant at the top of the file would be consistent with other datasets

Reply:
okay @Conchylicultor @us , I will add them in the top. :+1: